mesa/space.py

```diff
@@ -34,12 +37,44 @@ class Grid(object):
         get_cell_list_contents: Returns the contents of a list of cells ((x,y) tuples)
     '''
 
-    width = None
-    height = None
-    torus = False
-    grid = None
     default_val = lambda s: None
 
+
+    class GridOccupiedIter:
+        """
+        Return just the (un)occupied cells of the grid.
```

Comment:
@gcallah -- for someone reading this -- what form does the grid need to be passed in? Can you add that and/or an example grid to the doc string?